source/includes/configuration/_02-get-historical-snapshot.md.erb

```diff
@@ -22,26 +28,23 @@ $ curl 'https://ci.example.com/go/api/admin/config/5059cf548db9ea2d1b9192b45529c
 </cruise>
 ```
 
-Gets the configuration with the specified version.
+Gets the current configuration file.
 
 <%= available_since('14.3.0') %>
 
 ### HTTP Request
 
-`GET /go/api/admin/config/:commit_sha.xml`
+`GET /go/api/admin/config/config.xml`
 
 **Other convenience APIs**
 
-`GET /go/api/admin/config/config.xml`
+`GET /go/api/admin/config/current.xml`
 
     or
 
-`GET /go/api/admin/config/current.xml`
+`GET /go/api/admin/config/:md5.xml`
```

Comment:
I am ok with keeping it, but it's a really wierd api, md5.xml? 
I understand why it was confused earlier with the git sha, because the diff api uses git shas.
is there any more info I need to put in this?